compiler/docs/src/components/Layout/section-css.js

```diff
@@ -0,0 +1,62 @@
+export default `
+	h1 {
```

Comment:
If there were a way to have this be actual css or a styled component, that might be easier to edit. It can be pretty easy to forget a semicolon when editing a lot of css as an unstyled string. E.g. maybe moving the whole Section component in here would cause it to highlight as css?